SU-SWS/stanford_basic
# D8CORE-631: added max-width for h2 and h3

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-631_
This adjusts the max-width for H2 and H3 to be proportional to the body text.

# Needed By (Date)
- Sprint end

# Urgency
- Fixes broken stuff

# Steps to Test

Checkout this branch
Create a page with:
- H2 and body text
- H2 centered and body text
- H3 and body text
Verify you like how it looks.

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-631

## Related PRs

## More Information
This problem occurs because of how we are limiting the number of characters per line.

## Folks to notify
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
